src/bci_build/package/appcontainers.py

```diff
@@ -2,19 +2,19 @@
 import os
 from typing import Dict
 
-from bci_build.package import _build_tag_prefix
-from bci_build.package import ALL_NONBASE_OS_VERSIONS
-from bci_build.package import ApplicationStackContainer
-from bci_build.package import BuildType
-from bci_build.package import CAN_BE_LATEST_OS_VERSION
-from bci_build.package import DOCKERFILE_RUN
-from bci_build.package import generate_disk_size_constraints
-from bci_build.package import OsContainer
-from bci_build.package import OsVersion
-from bci_build.package import Package
-from bci_build.package import PackageType
-from bci_build.package import Replacement
-from bci_build.package import SupportLevel
+from .dataclasses import ApplicationStackContainer
+from .dataclasses import OsContainer
+from .dataclasses import Package
+from .dataclasses import PackageType
+from .dataclasses import Replacement
+from .dataclasses import SupportLevel
```

Comment:
Please use a different name, this is imho ambiguous as the name clashes with the stdlib & pydantic dataclass module, but provides classes build on top of it and not those compatible with stdlib dataclasses

Reply:
changed to bciclasses.